src/Sulu/Bundle/AdminBundle/Resources/js/components/Tabs/variables.scss

```diff
@@ -1 +1,5 @@
+@import '../../containers/Application/colors.scss';
+
 $tabMenuHeight: 40px;
+$tabMenuBackgroundColor: $alto;
+$lightTabMenuBackgroundColor: $white;
```

Comment:
We usually start these variables with the name of the component, so that there is kind of a namespacing, so here it would be `tabMenuLightBackgroundColor`